src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs

```diff
@@ -30,6 +30,7 @@ public RestoreCommand(IApplicationEnvironment env)
             FileSystem = new PhysicalFileSystem(Directory.GetCurrentDirectory());
             MachineWideSettings = new CommandLineMachineWideSettings();
             ScriptExecutor = new ScriptExecutor();
+            FailedPackages = new Dictionary<string, List<string>>();
```

Comment:
add explicit comparer (`StringComparer.Ordinal`) here and on line 58 ?  (might not be as common in this repo)